elixir_buildpack.config

```diff
@@ -0,0 +1,6 @@
+# Elixir version
+elixir_version=1.11.4
+
+# Erlang version
+# available versions https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/otp-versions
+erlang_version=21.2.5
```

Comment:
Should we use the same version as in the tool version - `23.3` 🤔

Reply:
Fixed